underfs/s3/src/main/java/alluxio/underfs/s3/S3Utils.java

```diff
@@ -22,34 +22,42 @@
  */
 public final class S3Utils {
```

Comment:
Could the `*Utils` classes be consolidated? Though you do not need to refactor it in this PR.